lte/gateway/c/core/oai/tasks/amf/amf_app_main.cpp

```diff
@@ -0,0 +1,229 @@
+    /* Handle uplink NAS message Recevied from the UE */
+    case AMF_APP_UPLINK_DATA_IND:
+      OAILOG_DEBUG(LOG_AMF_APP, "UPLINK_NAS_MESSAGE received\n");
+      amf_app_handle_uplink_nas_message(
+          amf_app_desc_p, AMF_APP_UL_DATA_IND(received_message_p).nas_msg,
+          AMF_APP_UL_DATA_IND(received_message_p).ue_id,
+          AMF_APP_UL_DATA_IND(received_message_p).tai);
+      break;
+    /* Handle PDU session Response from UE */
+    case N11_CREATE_PDU_SESSION_RESPONSE:
+      OAILOG_DEBUG(LOG_AMF_APP, "PDU SESSION RESPONSE received\n");
+      OAILOG_DEBUG(
+          LOG_AMF_APP,
+          "session created in SMF for imsi:%s for session_id:%d \n",
+          N11_CREATE_PDU_SESSION_RESPONSE(received_message_p).imsi,
+          N11_CREATE_PDU_SESSION_RESPONSE(received_message_p).pdu_session_id);
+      amf_app_handle_pdu_session_response(
+          &N11_CREATE_PDU_SESSION_RESPONSE(received_message_p));
+      break;
+    case AMF_APP_SUBS_AUTH_INFO_RESP:
+      // response_p =
+      // &(received_message_p->ittiMsg.amf_app_subs_auth_info_resp); auth_info =
+      // &(response_p->auth_info);
+      amf_nas_proc_authentication_info_answer(
+          &AMF_APP_AUTH_RESPONSE_DATA(received_message_p));
+      break;
+
+    case AMF_IP_ALLOCATION_RESPONSE:
+      itti_amf_ip_allocation_response_t* response_p;
+
+      response_p = &(received_message_p->ittiMsg.amf_ip_allocation_response);
+
+      amf_smf_handle_ip_address_response(response_p);
+
+      break;
+
+    /* Handle PDU session resource setup response */
+    case NGAP_PDUSESSIONRESOURCE_SETUP_RSP:
+      /* This is non-nas message and can be handled directly to check if failure
+       * or success messages are coming from NGAP
+       */
+      OAILOG_DEBUG(
+          LOG_AMF_APP, "NGAP_PDU SESSION RESOURCE SETUP RESPONSE received\n");
+      amf_app_handle_resource_setup_response(
+          NGAP_PDUSESSIONRESOURCE_SETUP_RSP(received_message_p));
+      break;
+    /* Handle PDU session resource release response */
+    case NGAP_PDUSESSIONRESOURCE_REL_RSP:
+      /* This is non-nas message and can be handled directly to check if failure
+       * or success messages are coming from NGAP
+       */
+      OAILOG_DEBUG(LOG_AMF_APP, "NGAP_PDUSESSIONRESOURCE_REL_RSP received\n");
+      amf_app_handle_resource_release_response(
+          NGAP_PDUSESSIONRESOURCE_REL_RSP(received_message_p));
+      break;
+    case N11_NOTIFICATION_RECEIVED:
+      /* This case handles Notification Received for Paging or other events
+       * or success messages are coming from NGAP
+       */
+      OAILOG_INFO(LOG_AMF_APP, "N11_NOTIFICATION_RECEIVED received\n");
+      imsi64 = itti_get_associated_imsi(received_message_p);
+      OAILOG_INFO(
+          LOG_AMF_APP,
+          "imsi ===============> %lu PAGING NOTIFICATION =======> "
+          "message_type = %d \n",
+          imsi64, ITTI_MSG_ID(received_message_p));
+      // amf_defs.amf_app_handle_notification_received(
```

Comment:
remove if unused or comment if it is a placeholder for future.

Reply:
comment addressed